server/market/market.go

```diff
@@ -2589,16 +2589,29 @@ func (m *Market) processReadyEpoch(epoch *readyEpoch, notifyChan chan<- *updateS
 		log.Errorf("Error updating API data collector: %v", err)
 	}
 
+	basicMatchesSell := make(map[uint64]uint64)
+	basicMatchesBuy := make(map[uint64]uint64)
+	for _, matchSet := range matches {
+		for _, match := range matchSet.Matches() {
+			if match.Maker.Sell {
```

Comment:
`if match.Taker.Sell`.

I'm 99% certain that the buy/sell for a match considers the taker.

Reply:
Thanks, I was confused by this. Believe it is properly now.